editor/src/components/editor/actions/actions.tsx

```diff
@@ -2355,8 +2355,12 @@ export const UPDATE_FNS = {
             EP.isRootElementOfInstance(elementPath) &&
             EP.isParentOf(getElementPathFromInsertionPath(parentPath), elementPath),
         )
-        if (anyTargetIsARootElement && targetThatIsRootElementOfCommonParent == null) {
```

Comment:
`targetThatIsRootElementOfCommonParent == null` is this not necessary anymore?

Reply:
yes, this needs a real fix soon